engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -252,6 +260,9 @@ class WorkflowActor(val workflowId: WorkflowId,
     case Event(AbortWorkflowCommand, WorkflowActorData(Some(actor), _, _, _)) =>
       actor ! EngineLifecycleActorAbortCommand
       goto(WorkflowAbortingState)
+    case Event(EngineStatsActor.JobCountQuery, _) =>
+      sender ! EngineStatsActor.NoJobs
+      stay()
```

Comment:
My thinking here was that it was going to be more prone to stuff disappearing, timeouts, etc. However I moved the main block to unhandled so we'll see how it goes